index.html

```diff
@@ -231,6 +231,35 @@ <h1 style="margin: 0.2em 0;font-size: 3em;font-weight: 300;">Roadmap<a class=" s
                     and <a href="https://github.com/scalameta/scalameta/issues/359">#359</a>
                     on our issue tracker if that's important for you.
                 </p>
+                <div style="margin: 0px;color: #333;text-align: center;padding: 2.5em 2em 0;border-bottom: 1px solid #eee;display: block;" id="FAQ" class=" scalatex-site-Styles-hoverContainer scalatex-site-Styles-headerTag">
+                    <h1 style="margin: 0.2em 0;font-size: 3em;font-weight: 300;">FAQ
+                        <a class=" scalatex-site-Styles-headerLink" href="#FAQ" style="position: absolute;right: 0px;"><i class="fa fa-link"></i></a>
+                    </h1>
+                    <br />
+                </div>
+                <p>
+                    <b>Does Meta support X Dialect</b>
+                    <ul>
+                        <li>Scala 2.10.X: <b>No -> <a href="https://github.com/scalameta/scalameta/issues/295">#295</a></b></li>
```

Comment:
Yes now :) https://github.com/scalameta/sbthost including semantic support for sbt 0.13 .sbt files!